src/platform/Linux/NetworkCommissioningDriver.h

```diff
@@ -147,11 +150,14 @@ class LinuxWiFiDriver final : public WiFiDriver
         Platform::SharedPtr<Crypto::P256Keypair> clientIdentityKeypair;
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
     };
-
-    WiFiNetwork mSavedNetwork;
-    WiFiNetwork mStagingNetwork;
+    uint8_t currentNetId          = 0;
+    uint8_t connectedNetworkIndex = 0;
```

Comment:
Please follow existing mWhatever naming convention.

And please document what these members mean.  I can't tell from the name what currentNetId is, and in code below it seems to be the number of free slots in mStagingNetwork?  But that makes no sense with the other uses.  It's really impossible to review this PR to tell which part is wrong without this being clearly documented.  But some part is wrong.